HISTORY.rst

```diff
@@ -2,6 +2,13 @@
  History
 =========
 
+==========
+UNRELEASED
+==========
+* Upgrade Bootstrap and Bootswatch to v3.4.1. (`@vedranmiletic`_)
```

Comment:
Thanks for adding history -- form looks great! Your username needs to be defined blow at the bottom of the file like the others as build hits this issue:

```
WARNING: Unknown target name: "@vedranmiletic".
```